components/automate-ui/src/app/page-components/navbar/navbar.component.html

```diff
@@ -57,9 +57,9 @@
         <div role="menuitem" class="nav-link"><a routerLink="/settings" routerLinkActive="active" tabindex="0">Settings</a></div>
       </app-authorized>
     </div>
-    <div class="right-nav" role="menubar">
-      <app-projects-filter></app-projects-filter>
-      <app-profile></app-profile>
+    <div class="right-nav">
+      <app-projects-filter role="menuitem"></app-projects-filter>
```

Comment:
this isnt really a menu itself, though it maybe is a menuItem for the entire top nav 🤷